src/System.IO.FileSystem/tests/FileInfo/Open.cs

```diff
@@ -12,6 +12,40 @@ protected override FileStream CreateFileStream(string path, FileMode mode)
         {
             return new FileInfo(path).Open(mode);
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "FileInfo.Open(string, filemode) on netfx always uses FileAccess.ReadWrite instead of choosing a FileAccess based on the FileMode. This bug was fixed in netcoreapp.")]
```

Comment:
Should we open an issue to track this difference and mark it as netfx-port-consider?